tests/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+    "video": false,
+    "baseUrl": "http://cvat_ui"
```

Comment:
I would suggest setup default baseUrl to "http://localhost:3000", create another file (like ``cypress_ci.json``) and add parameter ``--config-file=cypress_ci.json`` to the command on CI. In this case users do not need change something by default (like described in ``CONTRIBUTING.md``)

https://docs.cypress.io/guides/references/configuration.html#Options

Reply:
Yes, I agree, thanks. This is the more correct way. I'll do it now.